packages/select-menu/src/index.tsx

```diff
@@ -115,6 +122,10 @@ class SelectMenuC<T extends OptionBase> extends React.Component<
   }
   isOptionSelected = (option: string) => {
     if (!this.props.isMultiSelect) {
+      let el = document.getElementById(this.props.value as any)
+      if (el) {
+        this.menuListRef.current.scrollTo(0, el.offsetTop)
```

Comment:
this logic ends up in scrolling each time that component is rendered, which results in jumping